### DISCUSSION TITLE
Faikin

### DISCUSSION THREAD

**Discussion by revk** [revk/xESP32-Faikout#27] [Announcements]
As you may have noticed I have completed the change from Daikin to Faikin.

I want to avoid any trademark issues. And it makes a nice joke.

Sorry for any inconvenience. ESP32 s/w upgrades should switch over cleanly.

**Comment by mhackdo18** [comment]
Interested to buy at your amazon, whats included in box. does it have wire connector from daikin a/c to esp32

does it compatible with all daikin model? mine is BRP072C42 is replace model of BRP072A42

**Reply by revk** [reply]
No, just the board, you need the S21 or X50 wiring. I have tested with two modules, so not idea if compatible with something else, sorry.
